docs/customization/account-portal/overview.mdx

```diff
@@ -52,6 +52,12 @@ The user profile page hosts the prebuilt [`<UserProfile />`](/docs/components/us
 
 Redirect your authenticated users to their user profile page using the [`<RedirectToUserProfile />`](/docs/components/control/redirect-to-userprofile) control component.
 
+### Unauthorized sign-in
+
+The unauthorized sign-in page doesn't host any prebuilt Clerk component. It displays a UI confirming that a session from an unrecognized device was successfully revoked. For more information, see the [Unauthorized sign-in](/docs/security/unauthorized-sign-in) feature.
```

Comment:
Are you sure this is a guide? I see it more as a page which explains what the feature is about.

Would it be confusing the page with the rest of the "Guides"?

![image](https://github.com/user-attachments/assets/391fa863-ce37-4b88-8ad2-15afca6db9ba)

Reply:
firstly, I'd say to update this suggestion with the original suggestion that accidentally didn't get applied:

```suggestion
The unauthorized sign-in page displays a UI confirming that a session from an unrecognized device was successfully revoked. For more information, refer to [the guide.](/docs/security/unauthorized-sign-in)
```

secondly, I'd call this a reference guide - it's kind of in the middle, but it's more so a reference for the "unauthorized sign in" feature, so we can call it a reference! this would update the suggestion to read:

```suggestion
The unauthorized sign-in page displays a UI confirming that a session from an unrecognized device was successfully revoked. For more information, refer to [the reference.](/docs/security/unauthorized-sign-in)
```

Reply:
i've applied the changes here [c36db71](https://github.com/clerk/clerk-docs/pull/1668/commits/c36db715cff89289a1cb8ba1f86db5f3b80f6a81)